helpers/index.js

```diff
@@ -5,11 +5,33 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+/**
+ * Converts arguments into a selenium readable format if test mode is set.
+ * @param {Array} args Arguments that should be converted
+ * @return {Array} converted arguments
+ */
+const convertLogArgs = (args) => {
+  if (window.e2eTestLogs) {
```

Comment:
To me, it makes more sense to call this global variable something like `window.testMode`